pubspec.lock

```diff
@@ -569,4 +569,4 @@ packages:
     version: "0.2.0+2"
 sdks:
   dart: ">=2.18.0 <3.0.0"
-  flutter: ">=3.0.0"
+  flutter: ">=2.10.0"
```

Comment:
Should we go back a version? In the previous iteration this was set to 3.0.0

Reply:
I'm not sure why or how that happened. I certainly don't remember reverting o.O